apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryDbType.cs

```diff
@@ -36,6 +36,18 @@ public enum BigQueryDbType
         /// </summary>
         Timestamp,
         /// <summary>
+        /// A civil date in the Gregorian calendar.
+        /// </summary>
+        Date,
+        /// <summary>
+        /// A civil time-of-day with up to microsecond precision.
```

Comment:
I think that *used* to be the official wording in the docs, but it's now just "with microsecond precision", so will match that.